lib/cmake_build_modules/add_visualizers.cmake

```diff
@@ -0,0 +1,2 @@
+message(STATUS "Adding target_sources() in add_visualizers.cmake")
```

Comment:
I moved this here, so the relative position of ``add_visualizers.cmake`` and ``nat.vis`` was the same.